task5/script.js

```diff
@@ -0,0 +1,41 @@
+const df = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  {
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+  },
+  {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+  }
+];
+
+function arrayPromise() {
+  return new Promise((resolve, reject) => {
+    resolve();
+  });
+}
+
+arrayPromise().then(() => {
```

Comment:
Take the list Item as a Promise object and if it succeeds, go to .then.

Reply:
Thank you for the review!